test/e2e/text.cy.js

```diff
@@ -0,0 +1,31 @@
+describe('Text', function () {
+  beforeEach(function () {
+    cy.getData().then(function (data) {
+      this.pages = data.contentObjects.filter(item => item._type === 'page' && item._classes !== 'assessment');
+      this.articles = data.filter(item => item._type === 'article');
+      this.blocks = data.filter(item => item._type === 'block');
+      this.components = data.filter(item => item._type === 'component');
```

Comment:
```suggestion
```
These are already available via helpful getters under `this.data`, so these lines can be deleted and the references below just need updated, e.g. `this.components` should be `this.data.components`